extension/js/newResponseNotification.js

```diff
@@ -1,63 +1,85 @@
-(function(){
-    let currentCount = document.getElementsByClassName("post has-profile").length;
-    let lastPost = document.getElementsByClassName("post has-profile")[currentCount -1]
+(function () {
+  let currentCount = document
```

Comment:
it has errors on pages like [search.php ](https://www.ivelt.com/forum/search.php?search_id=active_topics)

i think we should only run on the `viewtopic` page
```suggestion
    if (window.location.pathname !== '/forum/viewtopic.php') return ;

    let currentCount = document
```

Reply:
Agree. As I suggested in the first message of this PR:

- Check if we're on a topic page. Currently the script runs on any and every page (not only in this new version, this is an old issue).

Reply:
missed it sorry